src/pages/video-test.tsx

```diff
@@ -127,17 +128,14 @@ const EggheadPlayer: React.FC<{
                   {!isEmpty(cues) && <Tab>Notes</Tab>}
                   <Tab>Lessons</Tab>
                 </TabList>
-                <TabPanels
-                  id="notes-tab-scroll-container"
```

Comment:
I think that we want to have two separate scrollable containers - one for notes, second for lessons. So I removed scrolling behavior form the `TabPanels`